src/_pytest/nodes.py

```diff
@@ -528,7 +528,7 @@ def gethookproxy(self, fspath: "os.PathLike[str]"):
         warnings.warn(FSCOLLECTOR_GETHOOKPROXY_ISINITPATH, stacklevel=2)
         return self.session.gethookproxy(fspath)
 
-    def isinitpath(self, path: py.path.local) -> bool:
+    def isinitpath(self, path: "os.PathLike[str]") -> bool:
```

Comment:
Right, I forgot these existed when I changed `Session`. I would match the `Session` signature exactly:

```suggestion
    def isinitpath(self, path: Union[str, "os.PathLike[str]"]) -> bool:
```